code-of-conduct-ID.md

```diff
@@ -0,0 +1,40 @@
+## Pengantar
+
+Kami berdedikasi untuk memberikan pengalaman bebas pelecehan untuk semua orang, dan kami tidak mentolerir pelecehan peserta dalam bentuk apapun. Kami meminta Anda untuk memperhatikan orang lain dan bersikap profesional serta hormat kepada peserta lainnya. Kode dan prosedur terkait ini juga berlaku untuk perilaku yang tidak dapat diterima yang terjadi di luar lingkup kegiatan komunitas, di semua tempat komunitas— daring dan pribadi— serta komunikasi secara pribadi, dan dimanapun perilaku yang memiliki potensi untuk mempengaruhi keselamatan serta kesejahteraan anggota komunitas secara negatif. Peserta pameran, pembicara, sponsor, staf dan semua peserta lainnya pada acara yang diselenggarakan oleh Docker, Inc (DockerCon, *meetup*, *user group*) atau yang diadakan di fasilitas Docker, Inc tunduk pada Pedoman Komunitas dan Kode Etik ini.
+
+Keragaman dan inklusi membuat komunitas Docker kuat. Kami mendorong partisipasi dari latar belakang yang paling bervariasi dan beragam serta ingin menjadi sangat jelas tentang dimana kita berdiri.
```

Comment:
@edawine I will use `menjunjung tinggi` instead of just `menjunjung`. How about it?